### ISSUE TITLE
[CONFORMANCE] Expecting SUT to send a Client Confirm Active PDU.

### ISSUE THREAD

**Issue by ligurio** [FreeRDP/FreeRDP#5903]
**Describe the bug**

Some time ago Microsoft opened an RDP specifications and then published testsuites for testing RDP client implementations (conformance tests) - https://github.com/Microsoft/WindowsProtocolTestSuites. I ran tests in testsuite RDPBCGR against xfreerdp and many of them failed with similar errors (actually 68 tests out of 113):
```
2020-02-25 11:18:14.523 [Comment] Setting Server Capability.

2020-02-25 11:18:14.523 [Comment] SetServerCapability( supportServerBitmapCacheHost = True; supportFastPathInput = True; supportScancode = True; supportExtendedMouse = True; supportUnicodeKeyboard = True; supportAutoReconnect = True; supportRefreshRect = True; supportSuppressOoutput = True; supportVCComression = True; presentVCChunksize = True).

2020-02-25 11:18:14.523 [Comment] Sending a Server Demand Active PDU to SUT and advertise the support for the Bitmap Host Cache Support Capability Set.

2020-02-25 11:18:14.648 [Comment] Expecting SUT to send a Client Confirm Active PDU.

2020-02-25 11:18:14.648 [Debug] The bytes of the ErrorPDU: 03-00-02-24-02-F0-80-64-00-06-03-EB-70-82-15-15-02-13-00-EF-03-EA-03-01-00-EA-03-08-00-FD-01-46-52-45-45-52-44-50-00-13-00-00-00-01-00-18-00-04-00-07-00-00-02-00-00-00-00-15-04-00-00-00-00-00-00-01-01-02-00-1C-00-20-00-01-00-01-00-01-00-00-04-00-03-00-00-01-00-01-00-00-0E-01-00-00-00-03-00-58-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-01-00-14-00-00-00-01-00-00-00-AA-00-01-01-01-00-00-00-00-00-01-00-00-00-00-00-00-00-00-00-01-00-00-00-01-00-00-00-00-00-00-00-00-00-00-00-04-00-00-00-00-00-00-84-03-00-00-00-00-00-E9-FD-00-00-13-00-28-00-03-00-00-05-58-02-00-00-58-02-00-00-00-08-00-00-00-10-00-00-00-08-00-00-00-00-00-00-00-00-00-00-00-00-00-00-08-00-0A-00-01-00-14-00-14-00-0D-00-58-00-3D-00-00-00-09-04-00-00-04-00-00-00-00-00-00-00-0C-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-0F-00-08-00-02-00-00-00-10-00-34-00-FE-00-04-00-FE-00-04-00-FE-00-08-00-FE-00-08-00-FE-00-10-00-FE-00-20-00-FE-00-40-00-FE-00-80-00-FE-00-00-01-40-00-00-01-00-01-00-01-00-00-00-00-14-00-0C-00-00-00-00-00-40-06-00-00-0C-00-08-00-01-00-00-00-09-00-08-00-00-00-00-00-0E-00-08-00-01-00-00-00-05-00-0C-00-00-00-00-00-02-00-02-00-0A-00-08-00-06-00-00-00-07-00-0C-00-00-00-00-00-00-00-00-00-1B-00-06-00-01-00-1A-00-08-00-FF-FF-00-00-1C-00-0C-00-52-00-00-00-00-00-00-00-1D-00-49-00-01-12-2F-77-76-72-BD-63-44-AF-B3-B7-3C-9C-6F-78-86-03-31-00-31-00-00-00-01-00-00-00-25-00-00-00-C0-CB-08-00-00-00-01-00-C1-CB-1D-00-00-00-01-C0-CF-02-00-08-00-00-01-40-00-00-01-01-01-00-01-40-00-00-01-01-04

2020-02-25 11:18:14.648 [CheckFailed] Assert.IsInstanceOfType failed. Expected Type: <Microsoft.Protocols.TestTools.StackSdk.RemoteDesktop.Rdpbcgr.Client_Confirm_Active_Pdu>, Actual Type: <Microsoft.Protocols.TestTools.StackSdk.RemoteDesktop.Rdpbcgr.ErrorPdu>. Receiving Client_Confirm_Active_Pdu

2020-02-25 11:18:14.648 [Comment] at Microsoft.Protocols.TestSuites.Rdpbcgr.RdpbcgrAdapter.ExpectPacket[T](RdpbcgrServerSessionContext session, TimeSpan waitTimeSpan) in C:\agent\_work\6\b\TestSuites\RDP\Client\src\Adapter\RDPBCGR\RdpbcgrAdapter.cs:line 2213 at Microsoft.Protocols.TestSuites.Rdpbcgr.RdpbcgrAdapter.ExpectPacket[T](TimeSpan timeout) in C:\agent\_work\6\b\TestSuites\RDP\Client\src\Adapter\RDPBCGR\RdpbcgrAdapter.cs:line 2226

2020-02-25 11:18:14.663 [Debug] Microsoft.VisualStudio.TestTools.UnitTesting.AssertFailedException: Assert.IsInstanceOfType failed. Expected Type: <Microsoft.Protocols.TestTools.StackSdk.RemoteDesktop.Rdpbcgr.Client_Confirm_Active_Pdu>, Actual Type: <Microsoft.Protocols.TestTools.StackSdk.RemoteDesktop.Rdpbcgr.ErrorPdu>. Receiving Client_Confirm_Active_Pdu ===== Stack Trace ===== at Microsoft.Protocols.TestSuites.Rdpbcgr.RdpbcgrAdapter.ExpectPacket[T](RdpbcgrServerSessionContext session, TimeSpan waitTimeSpan) in C:\agent\_work\6\b\TestSuites\RDP\Client\src\Adapter\RDPBCGR\RdpbcgrAdapter.cs:line 2213 at Microsoft.Protocols.TestSuites.Rdpbcgr.RdpbcgrAdapter.ExpectPacket[T](TimeSpan timeout) in C:\agent\_work\6\b\TestSuites\RDP\Client\src\Adapter\RDPBCGR\RdpbcgrAdapter.cs:line 2226 === End of Stack Trace === at Microsoft.Protocols.TestTools.AsynchronousErrorProcessor.Process() at Microsoft.Protocols.TestTools.AsynchronousErrorProcessor.ReportAsyncException(Exception checkerException) at Microsoft.Protocols.TestTools.Checking.DefaultChecker`2.GenerateException(CheckResult checkResult, String message, Object[] parameters) at Microsoft.Protocols.TestTools.Checking.DefaultChecker`2.IsInstanceOfType(Object value, Type type, String message, Object[] parameters) at Microsoft.Protocols.TestSuites.Rdpbcgr.RdpbcgrAdapter.ExpectPacket[T](RdpbcgrServerSessionContext session, TimeSpan waitTimeSpan) in C:\agent\_work\6\b\TestSuites\RDP\Client\src\Adapter\RDPBCGR\RdpbcgrAdapter.cs:line 2212 at Microsoft.Protocols.TestSuites.Rdpbcgr.RdpbcgrAdapter.ExpectPacket[T](TimeSpan timeout) in C:\agent\_work\6\b\TestSuites\RDP\Client\src\Adapter\RDPBCGR\RdpbcgrAdapter.cs:line 2225

2020-02-25 11:18:14.663 [Comment] Trigger client to close all RDP connections for clean up.

2020-02-25 11:18:14.679 [Comment] RDP SUT Control Protocol Adapter: CommandId is 2: Success, agent: 10.16.170.60:4488.

2020-02-25 11:18:14.679 [Debug] The result of TriggerClientDisconnectAll is 1.

2020-02-25 11:18:14.679 [Comment] Stop RDP listening.

2020-02-25 11:18:14.679 [TestFailed] Microsoft.Protocols.TestSuites.Rdpbcgr.RdpbcgrTestSuite.S1_ConnectionTest_BasicSettingExchange_PositiveTest_V10Server Debug Trace: [PTF Internal Trace Log][2020-02-25 11:18:12.804] [Beacon Log Warning]: The log message will be cut down to 1024 bytes by Beacon Log because it exceeds the max length of TSAP payload. [PTF Internal Trace Log][2020-02-25 11:18:13.320] [Beacon Log Warning]: The log message will be cut down to 1024 bytes by Beacon Log because it exceeds the max length of TSAP payload. [PTF Internal Trace Log][2020-02-25 11:18:13.320] [Beacon Log Warning]: The log message will be cut down to 1024 bytes by Beacon Log because it exceeds the max length of TSAP payload. [PTF Internal Trace Log][2020-02-25 11:18:13.476] [Beacon Log Warning]: The log message will be cut down to 1024 bytes by Beacon Log because it exceeds the max length of TSAP payload. [PTF Internal Trace Log][2020-02-25 11:18:13.476] [Beacon Log Warning]: The log message will be cut down to 1024 bytes by Beacon Log because it exceeds the max length of TSAP payload. [PTF Internal Trace Log][2020-02-25 11:18:13.476] [Beacon Log Warning]: The log message will be cut down to 1024 bytes by Beacon Log because it exceeds the max length of TSAP payload. [PTF Internal Trace Log][2020-02-25 11:18:13.476] [Beacon Log Warning]: The log message will be cut down to 1024 bytes by Beacon Log because it exceeds the max length of TSAP payload. [PTF Internal Trace Log][2020-02-25 11:18:13.476] [Beacon Log Warning]: The log message will be cut down to 1024 bytes by Beacon Log because it exceeds the max length of TSAP payload. [PTF Internal Trace Log][2020-02-25 11:18:13.476] [Beacon Log Warning]: The log message will be cut down to 1024 bytes by Beacon Log because it exceeds the max length of TSAP payload. [PTF Internal Trace Log][2020-02-25 11:18:14.538] [Beacon Log Warning]: The log message will be cut down to 1024 bytes by Beacon Log because it exceeds the max length of TSAP payload. [PTF Internal Trace Log][2020-02-25 11:18:14.663] [Beacon Log Warning]: The log message will be cut down to 1024 bytes by Beacon Log because it exceeds the max length of TSAP payload. [PTF Internal Trace Log][2020-02-25 11:18:14.663] [Beacon Log Warning]: The log message will be cut down to 1024 bytes by Beacon Log because it exceeds the max length of TSAP payload. [PTF Internal Trace Log][2020-02-25 11:18:14.679] [Beacon Log Warning]: The log message will be cut down to 1024 bytes by Beacon Log because it exceeds the max length of TSAP payload.
```

**To Reproduce**
1. Setup test stand as described in documentation: https://github.com/Microsoft/WindowsProtocolTestSuites/blob/staging/TestSuites/RDP/Client/docs/RDP_ClientUserGuide.md, https://github.com/Microsoft/WindowsProtocolTestSuites/blob/staging/TestSuites/RDP/Client/docs/RDP_Overview_ClientTestDesignSpecification.md
2. Build XFreeRDP client and run [Python wrapper](https://github.com/microsoft/WindowsProtocolTestSuites/tree/staging/TestSuites/RDP/RDPSUTControlAgent/Python) on Linux machine. `settings.ini`:
```
[general]
bind_ip_adress = 0.0.0.0
bind_port = 4488
buffer_size = 1024

[client]
# The command to create a negotiate-based rdp session.
# {{address}} must be present, they will be replaced with the actual address and port by the agent.
Negotiate = /home/sergeyb/backup_sdb/FreeRDP/build/client/X11/xfreerdp /t:RDPClient /cert-ignore /rfx /u:Administrator /p:Password01! /v:{{ address }}
# The command to create a negotiate-based fullscreen rdp session.
# {{address}} must be present, they will be replaced with the actual address and port by the agent.
NegotiateFullScreen = /home/sergeyb/backup_sdb/FreeRDP/build/client/X11/xfreerdp /t:RDPClient /f /rfx /u:Administrator /p:Password01! /v:{{ address }}
# The command to create a CredSSP rdp session.
# {{address}} must be present, they will be replaced with the actual address and port by the agent
DirectCredSSP = /home/sergeyb/backup_sdb/FreeRDP/build/client/X11/xfreerdp /t:RDPClient /rfx /u:Administrator /p:Password01! /v:{{ address }}
# The command to create a CredSSP fullscreen rdp session.
# {{address}} must be present, they will be replaced with the actual address and port by the agent
DirectCredSSPFullScreen = /home/sergeyb/backup_sdb/FreeRDP/build/client/X11/xfreerdp /t:RDPClient /f /rfx /u:Administrator /p:Password01! /v:{{ address }}
```
3. Start testing on Windows machine
4. See failed tests in a report generated by Windows Test Manager

**Actual behavior**
Assert failed: `2020-02-25 11:18:14.648 [CheckFailed] Assert.IsInstanceOfType failed. Expected Type: <Microsoft.Protocols.TestTools.StackSdk.RemoteDesktop.Rdpbcgr.Client_Confirm_Active_Pdu>, Actual Type: <Microsoft.Protocols.TestTools.StackSdk.RemoteDesktop.Rdpbcgr.ErrorPdu>. Receiving Client_Confirm_Active_Pdu`

**Expected behavior**
All tests are passed

**Application details**
* Version: FreeRDP dc89923f4a3db2c5ae5d169ea80ba4d0ad29e899
* output of `/buildconfig`:

```
This is FreeRDP version 2.0.0-dev5 (dc89923f4)
Build configuration: BUILD_TESTING=OFF BUILTIN_CHANNELS=ON HAVE_AIO_H=1 HAVE_EXECINFO_H=1 HAVE_FCNTL_H=1 HAVE_INTTYPES_H=1 HAVE_MATH_C99_LONG_DOUBLE=1 HAVE_POLL_H=1 HAVE_PTHREAD_MUTEX_TIMEDLOCK=ON HAVE_PTHREAD_MUTEX_TIMEDLOCK_LIB=1 HAVE_PTHREAD_MUTEX_TIMEDLOCK_SYMBOL= HAVE_SYSLOG_H=1 HAVE_SYS_EVENTFD_H=1 HAVE_SYS_FILIO_H= HAVE_SYS_MODEM_H= HAVE_SYS_SELECT_H=1 HAVE_SYS_SOCKIO_H= HAVE_SYS_STRTIO_H= HAVE_SYS_TIMERFD_H=1 HAVE_TM_GMTOFF=1 HAVE_UNISTD_H=1 HAVE_XI_TOUCH_CLASS=1 WITH_ALSA=ON WITH_CAIRO=OFF WITH_CCACHE=ON WITH_CHANNELS=ON WITH_CLANG_FORMAT=ON WITH_CLIENT=ON WITH_CLIENT_AVAILABLE=1 WITH_CLIENT_CHANNELS=ON WITH_CLIENT_CHANNELS_AVAILABLE=1 WITH_CLIENT_COMMON=ON WITH_CLIENT_INTERFACE=OFF WITH_CUPS=OFF WITH_DEBUG_ALL=OFF WITH_DEBUG_CAPABILITIES=OFF WITH_DEBUG_CERTIFICATE=OFF WITH_DEBUG_CHANNELS=OFF WITH_DEBUG_CLIPRDR=OFF WITH_DEBUG_DVC=OFF WITH_DEBUG_KBD=OFF WITH_DEBUG_LICENSE=OFF WITH_DEBUG_MUTEX=OFF WITH_DEBUG_NEGO=OFF WITH_DEBUG_NLA=OFF WITH_DEBUG_NTLM=OFF WITH_DEBUG_RAIL=OFF WITH_DEBUG_RDP=OFF WITH_DEBUG_RDPDR=OFF WITH_DEBUG_RDPEI=OFF WITH_DEBUG_RDPGFX=OFF WITH_DEBUG_REDIR=OFF WITH_DEBUG_RFX=OFF WITH_DEBUG_RINGBUFFER=OFF WITH_DEBUG_SCARD=OFF WITH_DEBUG_SND=OFF WITH_DEBUG_SVC=OFF WITH_DEBUG_SYMBOLS=OFF WITH_DEBUG_THREADS=OFF WITH_DEBUG_TIMEZONE=OFF WITH_DEBUG_TRANSPORT=OFF WITH_DEBUG_TSG=OFF WITH_DEBUG_TSMF=OFF WITH_DEBUG_WND=OFF WITH_DEBUG_X11=OFF WITH_DEBUG_X11_CLIPRDR=OFF WITH_DEBUG_X11_LOCAL_MOVESIZE=OFF WITH_DEBUG_XV=OFF WITH_DSP_EXPERIMENTAL=OFF WITH_DSP_FFMPEG=OFF WITH_EVENTFD_READ_WRITE=1 WITH_FAAC=OFF WITH_FAAD2=OFF WITH_FFMPEG=TRUE WITH_FFMPEG=TRUE WITH_GFX_H264=ON WITH_GPROF=OFF WITH_GSM=OFF WITH_GSSAPI=OFF WITH_GSTREAMER_0_10=OFF WITH_GSTREAMER_1_0=ON WITH_ICU=OFF WITH_IPP=OFF WITH_JPEG=OFF WITH_LAME=OFF WITH_LIBRARY_VERSIONING=ON WITH_LIBSYSTEMD=OFF WITH_MACAUDIO=OFF WITH_MACAUDIO=OFF WITH_MACAUDIO_AVAILABLE=0 WITH_MANPAGES=ON WITH_MBEDTLS=OFF WITH_OPENCL=OFF WITH_OPENH264=OFF WITH_OPENSLES=OFF WITH_OPENSSL=ON WITH_OSS=ON WITH_PCSC=OFF WITH_PROFILER=OFF WITH_PULSE=OFF WITH_SAMPLE=OFF WITH_SANITIZE_ADDRESS=OFF WITH_SANITIZE_ADDRESS_AVAILABLE=1 WITH_SANITIZE_MEMORY=OFF WITH_SANITIZE_MEMORY_AVAILABLE=1 WITH_SANITIZE_THREAD=OFF WITH_SANITIZE_THREAD_AVAILABLE=1 WITH_SERVER=OFF WITH_SERVER_INTERFACE=ON WITH_SMARTCARD_INSPECT=OFF WITH_SOXR=OFF WITH_SSE2=ON WITH_SWSCALE=OFF WITH_THIRD_PARTY=OFF WITH_VAAPI=OFF WITH_VALGRIND_MEMCHECK=OFF WITH_VALGRIND_MEMCHECK_AVAILABLE=1 WITH_WAYLAND=ON WITH_WINPR_TOOLS=ON WITH_X11=ON WITH_X264=OFF WITH_XCURSOR=ON WITH_XEXT=ON WITH_XFIXES=ON WITH_XI=ON WITH_XINERAMA=ON WITH_XKBFILE=ON WITH_XRANDR=ON WITH_XRENDER=ON WITH_XSHM=ON WITH_XV=ON WITH_ZLIB=ON
Build type:          Release
CFLAGS:               -fPIC -Wall -Wno-unused-result -Wno-unused-but-set-variable -Wno-deprecated-declarations -fvisibility=hidden -Wimplicit-function-declaration -Wredundant-decls -DWINPR_DLL
Compiler:            GNU, 7.4.0
Target architecture: x64
```

**Desktop (please complete the following information):**
 - OS: Ubuntu Linux, Windows 7 Enterprise SP1 x86_64
```
~$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.4 LTS
Release:        18.04
Codename:       bionic
```

Protocol Test Manager 3.20.1.0
MS RDP Client Endpoint Test Suite 3.20.1.0
MS Protocol Test Framework 1.0.7500.0

**Additional context**

RDPBCGR specification
https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpbcgr/5073f4ed-1e93-45e1-b039-6e30c385867c

**Comment by nfedera** [comment]
@ligurio I assume the text in your bug description is a cut&paste error:

> xfreerdp and many of them failed with similar errors (actually 68 tests out of 113):

Shoudn't that be _1 test out ouf 113_ ? 

More specifically, it is the test called _S1_ConnectionTest_CapabilityExchange_PositiveTest_LargePointerCapSet_LargePointerNotSupport_ that fails - right ?

**Comment by ligurio** [comment]
@nfedera testsuite for RDPBCGR spec contains 113 tests and 68 of them fails with the same assert.
Failed testcases:
```
BVT_ConnectionTest_CapabilityExchange_PositiveTest
BVT_ConnectionTest_ConnectionFinalization_PositiveTest_BitmapHostCacheSupported
BVT_ConnectionTest_Disconnection_PositiveTest_ServerInitiated
BVT_ReactivationTest_PositiveTest_BitmapHostCacheSupported
S1_ConnectionTest_BasicSettingExchange_NegativeTest_InvalidEncryptionMethod
S1_ConnectionTest_BasicSettingExchange_PositiveTest_V10Server
S1_ConnectionTest_CapabilityExchange_PositiveTest_BitmapCacheHostSupportCapSet_NotPresent
S1_ConnectionTest_CapabilityExchange_PositiveTest_BitmapCapSet_24ColorDepth
S1_ConnectionTest_CapabilityExchange_PositiveTest_BitmapCapSet_DesktopResizeNotSupported
S1_ConnectionTest_CapabilityExchange_PositiveTest_BitmapCapSet_DrawingFlagsNotSet
S1_ConnectionTest_CapabilityExchange_PositiveTest_BitmapCapSet_MaxDesktopHeight
S1_ConnectionTest_CapabilityExchange_PositiveTest_BitmapCapSet_MaxDesktopWidth
S1_ConnectionTest_CapabilityExchange_PositiveTest_BitmapCodecsCapSet_BitmapCodecCount_LargeValue
S1_ConnectionTest_CapabilityExchange_PositiveTest_BitmapCodecsCapSet_BitmapCodecCount_MaxValue
S1_ConnectionTest_CapabilityExchange_PositiveTest_BitmapCodecsCapSet_NoBitmapCodec
S1_ConnectionTest_CapabilityExchange_PositiveTest_BitmapCodecsCapSet_NotPresent
S1_ConnectionTest_CapabilityExchange_PositiveTest_DesktopCompositionCapSet_CompositionNotSupport
S1_ConnectionTest_CapabilityExchange_PositiveTest_DesktopCompositionCapSet_NotPresent
S1_ConnectionTest_CapabilityExchange_PositiveTest_FontCapSet_NotPresent
S1_ConnectionTest_CapabilityExchange_PositiveTest_GeneralCapSet_FPOutputNotSupported
S1_ConnectionTest_CapabilityExchange_PositiveTest_GeneralCapSet_OSTypeUnspecified
S1_ConnectionTest_CapabilityExchange_PositiveTest_GeneralCapSet_OSTypeWindowsRT
S1_ConnectionTest_CapabilityExchange_PositiveTest_GeneralCapSet_RefreshRectNotSupported
S1_ConnectionTest_CapabilityExchange_PositiveTest_GeneralCapSet_SuppressOutputNotSupported
S1_ConnectionTest_CapabilityExchange_PositiveTest_InputCapSet_OnlySupportScanCodes
S1_ConnectionTest_CapabilityExchange_PositiveTest_LargePointerCapSet_LargePointerNotSupport
S1_ConnectionTest_CapabilityExchange_PositiveTest_LargePointerCapSet_NotPresent
S1_ConnectionTest_CapabilityExchange_PositiveTest_MultifragmentUpdateCapSet_MaxRequestSizeLargeValue
S1_ConnectionTest_CapabilityExchange_PositiveTest_MultifragmentUpdateCapSet_NotPresent
S1_ConnectionTest_CapabilityExchange_PositiveTest_OrderCapSet_ExtraFlagNotValid
S1_ConnectionTest_CapabilityExchange_PositiveTest_OrderCapSet_SolidPatternBrushOnly
S1_ConnectionTest_CapabilityExchange_PositiveTest_PointerCapSet_ColorPointerCacheSize_0
S1_ConnectionTest_CapabilityExchange_PositiveTest_PointerCapSet_PointerCacheSize_0
S1_ConnectionTest_CapabilityExchange_PositiveTest_SurfaceCommandsCapSet_NotPresent
S1_ConnectionTest_CapabilityExchange_PositiveTest_SurfaceCommandsCapSet_StreamSurfaceBitsNotSupported
S1_ConnectionTest_CapabilityExchange_PositiveTest_VirtualChannelCapSet_CompressionNotSupport
S1_ConnectionTest_CapabilityExchange_PositiveTest_VirtualChannelCapSet_MaxVCChunkSize
S1_ConnectionTest_CapabilityExchange_PositiveTest_VirtualChannelCapSet_VCChunkSizeNotPresent
S1_ConnectionTest_ConnectionFinalization_PositiveTest_BitmapHostCacheNotSupported
S1_ConnectionTest_ConnectionFinalization_PositiveTest_BitmapHostCacheSupported_sequenceTest_1
S1_ConnectionTest_ConnectionFinalization_PositiveTest_BitmapHostCacheSupported_sequenceTest_2
S1_ConnectionTest_ConnectionFinalization_PositiveTest_BitmapHostCacheSupported_sequenceTest_3
S1_ConnectionTest_Disconnection_NegativeTest_MCSDisconnectProviderUltimatum_InvalidTPKTLength
S10_FastPathOutput_PositiveTest_CachedPointer_WithColorPointer
S10_FastPathOutput_PositiveTest_CachedPointer_WithNewPointer
S10_FastPathOutput_PositiveTest_CachedPointer_WithNewPointer_MultiCachedPointers
S10_FastPathOutput_PositiveTest_CachedPointer_WithNewPointer_ReuseCacheIndex
S10_FastPathOutput_PositiveTest_ColorPointer_HotSpotOnTheBorderOfPointer
S10_FastPathOutput_PositiveTest_ColorPointer_InitPointer
S10_FastPathOutput_PositiveTest_ColorPointer_MaxAllowedCacheIndex
S10_FastPathOutput_PositiveTest_ColorPointer_MaxAllowedWidthHeightLargePointerNotSupported
S10_FastPathOutput_PositiveTest_ColorPointer_OddWidth
S10_FastPathOutput_PositiveTest_ColorPointer_UpdatePointer
S10_FastPathOutput_PositiveTest_NewPointer_HotSpotOnTheBorderOfPointer
S10_FastPathOutput_PositiveTest_NewPointer_InitPointer
S10_FastPathOutput_PositiveTest_NewPointer_MaxAllowedCacheIndex
S10_FastPathOutput_PositiveTest_NewPointer_MaxAllowedWidthHeightLargePointerNotSupported
S10_FastPathOutput_PositiveTest_NewPointer_OddWidth
S10_FastPathOutput_PositiveTest_NewPointer_UpdatePointer
S10_FastPathOutput_PositiveTest_NewPointer_XorBpp_32
S10_FastPathOutput_PositiveTest_PointerHidden
S10_FastPathOutput_PositiveTest_PointerPosUpdate_PointerMove
S10_FastPathOutput_PositiveTest_PointerPosUpdate_PointerOnTheBorderOfScreen
S2_ReactivationTest_PositiveTest_BitmapHostCacheNotSupported
S4_SlowPathInputTest_NegativeTest_ServerToClientSlowPath_InvalidEncryptFlag
S4_SlowPathInputTest_NegativeTest_ServerToClientSlowPath_InvalidMCSLength
S4_SlowPathInputTest_NegativeTest_ServerToClientSlowPath_InvalidTotalLenth
S4_SlowPathInputTest_NegativeTest_ServerToClientSlowPath_InvalidTPKTLength
```

I can publish test results with debug logs but a bit later.

**Comment by akallabeth** [comment]
@ligurio Maybe we should compare test configurations so we can reproduce your results here?
You have updated your test suite and are running the most recent version now?

**Comment by ligurio** [comment]
@akallabeth 

> You have updated your test suite and are running the most recent version now?

Yes, this time I used latest versions of test framework and testsuite (versions are specified in a description).

>  Maybe we should compare test configurations so we can reproduce your results here?

Sure. Published test report and PTM profile - https://drive.google.com/open?id=1Iaa34HUqhfAlvYKQcgLiIAhedUrEvVMW

**Comment by akallabeth** [comment]
@ligurio thank you. do you also have the log output of the `xfreerdp` run?

**Comment by ligurio** [comment]
@akallabeth Sure, added file with log to the same share.

**Comment by nfedera** [comment]
@ligurio  It seems that you are still using the outdated 3.19.3.0 version!

The `config\info.xml` in your `gh-5903-profile.ptm` (which is a inspectable zip file btw) is:

```xml
<ProfileInfo xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <TestSuiteName>RDP-Client</TestSuiteName>
  <Version>3.19.3.0</Version>
</ProfileInfo>
```
Before installing the rebuilt `RDP-TestSuite-ClientEP.msi` you must uninstall **all** _Microsoft RDP Client Endpoint Test Suite_ entries from Control Panel's _Uninstall or change a program_.

**Comment by nfedera** [comment]
@ligurio I've seen that you have changed the default value for the `RDP.Security.IsClientToServerEncrypted` property from to `true` to `false` ?
Why? Please change this to `true`again.

Also you should set the `IsWindowsImplementation` property that indicates that the SUT is a Windows Implementation to `false`.

**Comment by ligurio** [comment]
@nfedera Thanks!

I believe these issues were due to wrong conversion of profile by new version of Test Manager.
Anyway I set `RDP.Security.IsClientToServerEncrypted` to `true` and `IsWindowsImplementation` to `false` again and got one failed testcase `S1_ConnectionTest_CapabilityExchange_PositiveTest_LargePointerCapSet_LargePointerNotSupport` and one inconclusive testcase `S1_ConnectionTest_ConnectionInitiation_PositiveTest_HybridEXSelected`.

Test results and test log uploaded (`gh-5903-rerun1.log`, `gh-5903-rerun1-RDP-Client-2020-02-25-13-57-00.zip`).
